docs/pages/development/installation.md

```diff
@@ -117,34 +114,15 @@ Make the following changes to allow the `expo-dev-client` library to control pro
 <ConfigurationDiff source="/static/diffs/client/main-activity-and-application-expo-modules.diff" />
 </Tab>
 
-<Tab >
-
-Make the following changes to allow the `expo-dev-client` library to control project initialization in the **DEBUG** mode.
-
-<ConfigurationDiff source="/static/diffs/client/main-activity-and-application.diff" />
-</Tab>
-
 </Tabs>
 
 ## 3. Optional configuration
 
-There are a few more changes you can make to get the best experience, but you [can skip ahead to building](/development/getting-started/#building-and-installing-your-first-development-build) if you prefer.
-
-### Enable development with Expo CLI
-
-Expo CLI requires you to have the `expo` package installed so it can maintain consistent behavior in your project when new versions of the Expo SDK are released. The package will not be used directly by your project unless you import it in your application code, which is not recommended.
-
-<InstallSection packageName="expo" cmd={["npm install expo --save-dev"]} hideBareInstructions />
-
-### Disable packager autostart when building for iOS
-
-When you start your project on iOS, the metro bundler will be started automatically. This behavior might not be ideal when you want to use `expo start`. Our recommended solution is to remove the `Start Packager` action from building scripts. To do that you need to open the Xcode, go to "Build Phases" and remove the "Start Packager" action.
-
-<img src="/static/images/client/remove_start_packager.png" style={{maxWidth: "100%" }}/>
+There are a few more changes you can make to get the best experience, if you prefer.
 
 ### Add better error handlers
 
-Sometimes, for certain types of errors, we can provide more helpful error messages than the ones that ship by default with React Native. To turn this feature on you need to import `expo-dev-client` in your **index** file (in the managed workflow, you need to add this import on top of the **App.{js|tsx}**). Make sure that the import statement is above `import App from './App'`.
+Sometimes, for certain types of errors, we can provide more helpful error messages than the ones that ship by default with React Native. To turn this feature on, you need to import `expo-dev-client` in the entry point of your app (in the managed workflow, you need to add this import at the top of your **App.{js|tsx}**). Make sure that the import statement is above `import App from './App'`.
```

Comment:
I'm not sure why referencing managed workflow here is relevant. At the start of this doc, it is referencing the user to read the separate doc if they want to use `expo-dev-client` with managed workflow.

Reply:
agreed